mxabierto/adela
# featute: fix margin footer

### Changelog

* Adding space between  footer and the top button of each page from Adela system.

### How to test
* M:   app/assets/stylesheets/application.css.scss
* M:   app/views/admin/base/index.html.haml
* M:   app/views/admin/organizations/shared/_form.html.haml
* M:   app/views/admin/users/shared/_form.html.haml
* M:   app/views/catalog_versions/index.html.haml
* M:   app/views/catalog_versions/show.html.haml
* M:   app/views/catalogs/check.html.haml
* M:   app/views/devise/registrations/edit.html.haml
* M:   app/views/inventories/datasets/index.html.haml
* M:   app/views/inventories/datasets/new.html.haml
* M:   app/views/organizations/profile.html.haml
* M:   app/views/organizations/shared/_dashboard.html.erb

### Follow the next step:

a) Download the source.
b)  Start you local environment.

* How to validate:
In all the views in the system Adela you must have space between footer and the last button of each page.
